parlai/core/torch_agent.py

```diff
@@ -271,3 +275,34 @@ def parse(txt, splitSentences):
                 self.history['labels'] = labels
 
         return self.history['dialog']
+
+        def save(self, path):
```

Comment:
save should save the opt file as well, see e.g. seq2seq

Reply:
specifically, the opt should be saved in a separate file at modelfile.opt
let's also remove line 290 where we save it within the `model` dict

Reply:
oh good point. do you think it's fine that `load` doesn't load in the opt file? I saw in seq2seq it does so, but didn't know if I should put that in the parent class